modules/ROOT/pages/user-management.adoc

```diff
@@ -152,40 +152,327 @@ Each project can have multiple users with individual accounts allowing access to
-| View and open instances | {check-mark} | {check-mark} | {check-mark}
-| Access the Neo4j Customer Support Portal | {check-mark} | {check-mark} | {check-mark}
-| Perform all actions on instances footnote:[Actions include creating, deleting, pausing, resuming, and editing instances.] | {check-mark} | {check-mark} |
-| Clone data to new and existing instances | {check-mark} | {check-mark} |
-| Take on-demand snapshots | {check-mark} | {check-mark} |
-| Restore from snapshots | {check-mark} | {check-mark} |
-| Edit the project name | {check-mark} | |
-| Invite new users to the project | {check-mark} | |
-| Edit existing users' roles | {check-mark} | |
-| Delete existing users from the project | {check-mark} | |
-| View and edit billing information | {check-mark} | |
-|===
+The `metrics reader` role can be assigned to any user or service account.
+It has the same permissions as the `project viewer` role, but with some extra permissions specifically for reading metrics via an API endpoint.
+The role allows access to metrics for all instances in a project.
+Accessing metric endpoints requires xref:/api/authentication.adoc[Aura API Credentials] and the `metrics reader` role enables the creation of these credentials.
+
+The `metrics reader` role can view and open instances in the console, however, login to the instance is required to interact with it, with access to Explore and Query defined by the instance’s RBAC settings.
 
 [NOTE]
 ====
 Each project must have at least one Project Admin, but it is also possible for projects to have multiple Project Admins.
 ====
 
+:check-mark: icon:check[]
+
+.Roles and console capabilities
+[opts="header",cols="3,1,1,1,1,1"]
+|===
+| Capability
+| Viewer
+| Data Viewer
+| Metrics reader
+| Member
+| Admin
+
+| View users and their roles
+| {check-mark}
+| {check-mark}
+| {check-mark}
+| {check-mark}
+| {check-mark}
+
+| View and open instances
+| {check-mark}
+| {check-mark}
+| {check-mark}
+| {check-mark}
+| {check-mark}
+
+| Access the Neo4j Customer Support Portal
+| {check-mark}
+| {check-mark}
+| {check-mark}
+| {check-mark}
+| {check-mark}
+
+| Perform all actions on instances footnote:[Actions include creating, deleting, pausing, resuming, and editing instances.]
+|
+|
+|
+| {check-mark}
+| {check-mark}
+
+| Clone data to new and existing instances
+|
+|
+|
+| {check-mark}
+| {check-mark}
+
+| Take on-demand snapshots
+|
+|
+|
+| {check-mark}
+| {check-mark}
+
+| Restore from snapshots
+|
+|
+|
+| {check-mark}
+| {check-mark}
+
+| Edit the project name
+|
+|
+|
+|
+| {check-mark}
+
+| Invite new users to the project
+|
+|
+|
+|
+| {check-mark}
+
+| Edit existing users' roles
+|
+|
+|
+|
+| {check-mark}
+
+| Delete existing users from the project
+|
+|
+|
+|
+| {check-mark}
+
+| View and edit billing information
+|
+|
+|
+|
+| {check-mark}
+|===
+
+
+Users within a project can access instances seamlessly with their console role xref:security/tool-auth.adoc[Tool authentication with Aura user] is enabled.
```

Comment:
This isn't quite clear to me.  Is there something missing between `console role` and the `Tool authentication with Aura user` link?

Reply:
If Tool authentication is not enabled, a user's console role won't be mapped to a predefined role.

Reply:
ah ok. Thanks